homeassistant/components/motion_blinds/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "domain": "motion_blinds",
-  "name": "Motion Blinds",
```

Comment:
Looking into if this is safe to change or that other things need simultaneous adjustment.

Reply:
Good idea. I left the domain unchanged as I suspect that is the main identifier used and the name is only used as a display name. I think changing the domain too would create issues with the devices that are already configured not working anymore?

Reply:
True, the domain and folder name should not be changed.
Changing the name is okay, but we need to run hassfest to make sure the generated files are also updated

Reply:
I checked and the only file that needs to be adjusted is in homeassistant/generated/integrations.json.
Could you change the name there?
After that I can merge this.

Reply:
Done! I also corrected **Smart Blinds** to **Smartblinds**